src/reference/05-Faq/00.md

```diff
@@ -30,18 +30,25 @@ Frequently Asked Questions
 
 ### Project Information
 
-#### What does sbt stand for, and how should I spell it?
+#### What does the name "sbt" stand for and shouldn't it be written "SBT"?
 
-sbt is just sbt: It's no longer an acronym. When the project was first started
-it was called *Simple Build Tool*, but when the first release came out,
-it was already called [sbt 0.3.2](http://www.scala-lang.org/old/node/392.html).
+**TL;DR** the name sbt doesn't stand for anything, it's just "sbt", and it should be written that way.
 
-sbt is a proper noun that must be spelled with all lowercase.
-Any deviation from the guideline such as title casing or all caps are subject to side-eye frown face.
```

Comment:
Why was this taken out?

Reply:
Title casing is something I have to argue against even in-house.

Reply:
Didn't like it.